clientv3/client.go

```diff
@@ -182,6 +187,10 @@ func (c *Client) retryConnection(oldConn *grpc.ClientConn, err error) (*grpc.Cli
 // connection is established.
 func dialEndpointList(c *Client) (*grpc.ClientConn, error) {
 	var err error
+	if len(c.Endpoints()) == 0 {
```

Comment:
would it be better to put this in `New` instead of checking in the retrydialer every redial?